locales/en-US/snoozetabs.properties

```diff
@@ -1,35 +1,45 @@
-# LOCALIZATION NOTE: This add-on will let you select tabs to close for a while, after which they will re-appear in your browser with a notification.
+# This add-on will let you select tabs to close for a while, after which they
```

Comment:
Historically we used `LOCALIZATION NOTE (string_ID):`, the reality is that Pontoon shows the comment before the string, there's no actual way to bind the comment to a string (or a group of strings).